va/va.go

```diff
@@ -524,3 +540,25 @@ func (va VAImpl) fetchHTTP(identifier string, token string) ([]byte, string, *ac
 
 	return body, url.String(), nil
 }
+
+//this reverseaddr funntion is from net/dnsclient.go form golang
+func reverseaddr(addr string) string {
+	ip := net.ParseIP(addr)
+	if ip == nil {
+		return ""
+	}
+	// Apperently IP type in net package saves all ip in ipv6 formant, from biggest byte to smallest. we need last 4 bytes, so ip[15] to ip[12]
+	if ip.To4() != nil {
+		return fmt.Sprintf("%d.%d.%d.%d.in-addr.arpa.", ip[15], ip[14], ip[13], ip[12])
+	}
+	// Must be IPv6
+	buf := make([]string, 0, len(ip)+1)
+	// Add it, in reverse, to the buffer
+	// each part of slice has two quads, so manual hex conversion is needed and fmt %x doesn't help
```

Comment:
I think this comment about "fmt %x doesn't help" isn't accurate anymore and needs to be removed or updated. You're using @rolandshoemaker's suggestion and have `fmt.Sprintf("%x.%x", ...` in the loop body.